lua/lvim/core/nvimtree.lua

```diff
@@ -44,36 +51,61 @@ function M.config()
       renderer = {
         add_trailing = false,
         group_empty = false,
-        highlight_git = true,
-        full_name = false,
+        highlight_git = "none",
```

Comment:
@abzcoding, the original behavior was changed, so we'd need to select a value to enable this

```suggestion
        highlight_git = "none" | "icon" | "name" | "all" ,
```

Reply:
i'm using neotree , so not sure which one to choose, i would assume `icon` would be preferred

Reply:
`icon` wasn't so visible with tokyonight, so I went with `name` which is a bit more obvious in case there's a performance hit from the operation 😄